README.md

```diff
@@ -143,6 +147,20 @@ Edit the `.params` file and add the required settings.
 }
 ```
 
+* If the service credentials from IBM Watson Assistant is IAM based as below, populate the IAM apikey, url, and workspace_id and replace the `.params` file
```

Comment:
same thing here re: s/is/are/ and either IAM-based or more clearly say "contain IAM credentials..."

Reply:
```If the service credentials from IBM Watson Assistant are IAM based as shown in the diagram below, populate the IAM apikey, url, and workspace_id fields  and replace the `.params` file with the populated JSON ``` how about this @markstur ?